modules/icon/css/README.md

```diff
@@ -92,36 +162,72 @@ the 500 weight (e.g. `blueberry500`).
 
 A custom color value (e.g. `#FFFFFF`) may also be used.
 
-#### Accent Icons
+##### `data-color`
 
-Accent icons may be colored in 1 color.
+Set the color for fill and accent layers simultaneously. This can be overridden by `data-fill-color`
+and `data-accent-color`.
+
+### Hover Coloring
+
+Each of the 3 layers can also be assigned a hover color.
 
 ```html
-<i className="wdc-icon" data-icon="apple" data-category="accent" data-color="cinnamon300" />
+<i
+  className="wdc-icon"
+  data-icon="activityStream"
+  data-category="system"
+  data-hover-fill-color="blueberry600"
+  data-hover-accent-color="blueberry400"
+  data-hover-background-color="blueberry200"
+/>
+```
 
-<-- Shorthand Coloring -->
-<i className="wdc-icon" data-icon="apple" data-category="accent" data-color="chiliMango" />
+You may use the `data-hover-color` shorthand to set the fill and accent hover colors simultaneously.
+
+```html
+<i
+  className="wdc-icon"
+  data-icon="activityStream"
+  data-category="system"
+  data-accent-color="blueberry500"
+  data-hover-color="cinnamon300"
+/>
 ```
 
-**`data-color`**
+##### `data-hover-fill-color`, `data-hover-accent-color`, `data-hover-background-color`
 
-See [System Icons coloring](#system-icons).
+Set the hover color of the corresponding layer.
 
-#### Applet Icons
+##### `data-hover-color`
 
-Applet icons are colored by hues (e.g. blueberry). Selecting a hue will apply the proper color
-shades to the applet.
+Set the hover color for fill and accent layers simultaneously. This can be overridden by
+`data-hover-fill-color` and `data-hover-accent-color`.
+
+### System Icon Circles
 
 ```html
-<i className="wdc-icon" data-icon="rocket" data-category="applet" data-color="cinnamon" />
+<i
+  className="wdc-icon"
+  data-icon="activityStream"
+  data-category="system"
+  data-circle="true"
```

Comment:
Can we just assume this if `data-circle-background` is defined and remove the default?

Reply:
Just to clarify: `data-circle="true"` on its own will render the circle with its default background color, but you can also do `data-circle-background="COLOR"` on its own (**without** `data-circle="true"`) to render the circle in a non-default color. Is that correct?

Reply:
Ah - should have looked closer at the logic. Either way, I would consolidate. You could just use `data-circle-background="true"` to get the default, and a canvas color, or custom hex to get the non-default.

Reply:
Got it, will do.